pandas/core/arrays/datetimes.py

```diff
@@ -973,6 +1088,35 @@ def to_perioddelta(self, freq):
         m8delta = i8delta.view('m8[ns]')
         return TimedeltaArrayMixin(m8delta)
 
+    def astype(self, dtype, copy=True):
```

Comment:
maybe put in Array-Like Methods section?

Reply:
It's also part of the ExtensionArray interface.

Reply:
Ah, though I see your comment on `.view` now, and yeah these should probably be co-located...